src/client/components/shared.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+import styled from "styled-components";
+
+/*************************************************************************/
+
+export const GenericButton = styled.button`
+  max-width: 200px;
```

Comment:
Instead of using px convert to either strictly rem or em. Change globally

Reply:
Let's discuss this on Thursday also. Unsure about whether we even need a shared.js file since we might be fine using material-ui components